### PR TITLE
dev/mailing#59 Update the version of zetacomponents/mail package to b…

### PR DESCRIPTION
**Pull request by seamuslee001**
…e one that potentially will support XAUTH2 for google purposes

Overview
----------------------------------------
This updates the version of zetacomponents/mail following my PR https://github.com/zetacomponents/Mail/pull/78 being merged in

ping @eileenmcnaughton @demeritcowboy @mfb

### CODE REVIEW

**Comment by mfb** [comment]
@seamuslee001 can you instead update it to v1.9.0 or is a specific git commit still needed?

File: composer.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "tecnickcom/tcpdf" : "6.2.*",
     "totten/ca-config": "~17.05",
     "zetacomponents/base": "1.9.*",
-    "zetacomponents/mail": "dev-master",
+    "zetacomponents/mail": "1.9.*",
```
Comment (mfb) [inline composer.json]: Theoretically we could use ^1.9 here? Because later 1.x releases won't break compatibility - unless they do :)

Comment (seamuslee001) [reply composer.json]: in theory yes but just figured go for the simplest approach right,

Comment (colemanw) [reply composer.json]: If we did that wouldn't we also need to do that for "zetacomponents/base" as well?
I'm happy to merge this PR and have that be a separate discussion.

Comment (mfb) [reply composer.json]: Sure go ahead and merge.

Just thinking that it's best to not be overly restrictive on updates until you know there is (or is likely to be) a problem.

Comment (mfb) [reply composer.json]: Looks like zetacomponents/mail:1.9.0 requires zetacomponents/base:~1.8"

**Comment by seamuslee001** [comment]
test fail unrelated

**Pull request merged**